README.Rmd

```diff
@@ -555,7 +566,7 @@ The Cook County property tax system is complex and can sometimes create perverse
 
 For example, most property owners want their property taxes to be as low as possible, and are thus disincentivized from reporting characteristic errors which could raise their assessed value. Conversely, if a property owner plans to sell their home on a listing website, then they have a strong incentive (the highest possible sale price) to ensure the website accurately reflects their property's characteristics. Listing websites know this and offer easy ways to self-update property attributes.
 
-Additionally, the assessment system has a built-in number of "hard boundaries" such as property class and assessor neighborhood. These boundaries create the (mis)perceived incentive to slightly alter the characteristics of marginal properties. For example, subtracting 10 square feet from a property's total footprint in order to end up in a different property class. In actuality, property class has no impact on predictions from our models.
+Additionally, the assessment system has a built-in number of "hard boundaries" such as property class and assessor neighborhood. These boundaries create the (mis)perceived incentive to slightly alter the characteristics of marginal properties. For example, subtracting 10 square feet from a property's total footprint in order to end up in a different property class. In actuality, property class has almost no impact on predictions from our models.
```

Comment:
It has an impact, just not as much of one as people would probably think. Square footage and location swamp most of the smaller characteristic effects.

Reply:
To test this, I used the API on a Rogers PIN: a class 203 with 1045 sf, and an original predicted value of $308k.

|     | sf: 1045 (original) | sf: 999 (change) |
| ------------------- | ------------------- | ----------------- |
| class: 203 (original) | $308k (original) | $304k | 
| class: 202 (change) | $293k | $286k | 

In the context of SHAPs, SMD, and the single randomly selected PIN above, I think it's not accurate to describe this as "almost no impact." It's not massive, and in this example it's not enough to meet Valuations' advice of a 10% change in valuation to justify a change on appeal, but I wouldn't describe it as almost nothing. 

It looks like the context of this section is about perverse incentives to not disclose accurate information, and the section actually reads quite well with this paragraph eliminated. Alternatively, if this is a point you really want to make, I suggest this:

Additionally, the assessment system has a built-in number of "hard boundaries." One example is property class code, a code that groups together characteristics like square footage and age. The hard boundary of class code creates the perception that if a property changes class code, for example by subtracting 10 square feet from a property's building footprint, this will result in a substantially lower assessment and lower tax bill. In actuality, our model's prediction depends highly on the actual square footage, age, and multiple location factors, in addition to class code and other features, to predict property values.

Reply:
Alight, I just dropped the whole paragraph in 0b54998